_targets/build.project.armv7m7-imxrt105x

```diff
@@ -55,13 +55,15 @@ KERNEL_PHBASE=$((0x0)) # itcm start address
 KERNEL_PHOFFS=$((0x0))
 export KERNEL_PHADDR=$(printf "%08x" $((KERNEL_PHBASE + KERNEL_PHOFFS)))
 
+# All data/code maps need to be cacheable to allow unaligned accesses
 
 # Pre-init script is launched before user script
 PREINIT_SCRIPT=(
 	"map itcm 0x0 0x28000 rwx"
 	"map dtcm 0x20000000 0x20048000 rw"
-	"map ocram 0x20200000 0x20220000 rwxc"
-	"map xip1 0x60000000 0x60800000 rx"
+	"map ocram 0x20200000 0x20220000 rwxcb"
+	"map xip1 0x60000000 0x60800000 rxcb"
+	"map io 0x40000000 0x50000000 rx"
```

Comment:
Shouldn't it be `rw`?
```suggestion
	"map io 0x40000000 0x50000000 rw"
```

Reply:
Is this really fixed?

Reply:
Somehow I forgot/did not stage changes, it's fixed now